### ISSUE TITLE
H98解説の第2段落について

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#1584]
A11Y Slackコメントより

https://waic.jp/docs/WCAG21/Techniques/html/H98

> > 認知障害のある利用者がよりアクセシブルになる
>
> この訳だと利用者自身がアクセシブルになるように読めてしまうので、「認知障害のある利用者にとってよりアクセシブルになる」などのほうがいいんじゃないかと。
> 原文は
> > make them more accessible to people with cognitive disabilities.

原文：
https://www.w3.org/WAI/WCAG21/Techniques/html/H98

\# ところで、H98のW3C原文、Date: Updated 24 December 2022.とあって結構最近に更新されていますね…

**Comment by momdo** [comment]
こういうことでしょうか？

現行訳：
> 入力がプログラムによって割り当てられている場合、サードパーティのプラグイン及びソフトウェアがこれらのフォームフィールドを操作して、認知障害のある利用者がよりアクセシブルになる。

原文：
> When inputs have been programmatically assigned, third party plugins and software can manipulate these form fields to make them more accessible to people with cognitive disabilities. 

案：
入力がプログラムによって割り当てられている場合、サードパーティのプラグイン及びソフトウェアは、フォームフィールドを認知障害のある人にとってよりアクセシブルにするために、そのフォームフィールドを操作できる。

**Comment by bakera** [comment]
今回のご指摘とは異なる点なのですが、「入力がプログラムによって割り当てられている」というのも誤解を招きそうな感じがしますね……。

この解説は「達成基準 1.3.5: 入力目的の特定」(Identify Input Purpose) に関連するものとされています。この解説における input / inputs は達成基準1.3.5の input と同じ意味、すなわち「入力目的」と言っている文脈での「入力」です。"inputs have been programmatically assigned" というのは、その入力欄がプログラムによって識別されて目的を割り当てられている状態を指しているのであって、プログラムが値を入れているという話ではありません。

「入力がプログラムによって割り当てられている」という表現だと、「プログラムが値を自動入力している」というように読めてしまって、混乱を招くような気がしますがどうでしょうか……。

ここの input は「入力欄」とか「入力項目」くらいのほうがいいのかもしれません。
とはいえこのコメントはちょっと今回の趣旨とは離れるので、上記はそれはそれでOKかと思います。
